services/blockchain-connector/shared/sdk/tokens.js

```diff
@@ -94,7 +94,19 @@ const getTokenInitializationFees = async () => {
 	}
 };
 
+const hasUserAccount = async ({ address, tokenID }) => {
+	try {
+		return invokeEndpoint('token_hasUserAccount', { address, tokenID });
+	} catch (err) {
+		if (err.message.includes(timeoutMessage)) {
+			throw new TimeoutException('Request timed out when calling \'hasUserAccount\'.');
+		}
+		throw err;
+	}
+};
+
 module.exports = {
+	checkUserAccountExistsForToken: hasUserAccount,
```

Comment:
Please try to keep the method names as close and possibly short to the original method.
Please make relevant changes everywhere applicable.
```suggestion
	hasTokenUserAccount: hasUserAccount,
```

Reply:
renamed to `tokenHasUserAccount`